lua/supermaven-nvim/binary/binary_fetcher.lua

```diff
@@ -112,7 +112,14 @@ end
 
 function BinaryFetcher:local_binary_parent_path()
   local home_dir = self.homedir
```

Comment:
Won't this value always be `nil`?

Reply:
i undo my change to the initialization of self.homedir